modules/swagger-codegen/src/main/resources/swift5/modelObject.mustache

```diff
@@ -0,0 +1,82 @@
+
+public struct {{classname}}: Codable {
+
+{{#allVars}}
+{{#isEnum}}
+{{> modelInlineEnumDeclaration}}
+{{/isEnum}}
+{{/allVars}}
+{{#allVars}}
+{{#isEnum}}
+    {{#description}}/** {{description}} */
+    {{/description}}public var {{name}}: {{{datatypeWithEnum}}}{{^required}}?{{/required}}{{#defaultValue}} = {{{defaultValue}}}{{/defaultValue}}
+{{/isEnum}}
+{{^isEnum}}
+    {{#description}}/** {{description}} */
+    {{/description}}public var {{name}}: {{{datatype}}}{{^required}}?{{/required}}{{#defaultValue}} = {{{defaultValue}}}{{/defaultValue}}{{#objcCompatible}}{{#vendorExtensions.x-swift-optional-scalar}}
+    public var {{name}}Num: NSNumber? {
+        get {
+            return {{name}}.map({ return NSNumber(value: $0) })
+        }
+    }{{/vendorExtensions.x-swift-optional-scalar}}{{/objcCompatible}}
+{{/isEnum}}
+{{/allVars}}
+
+{{#hasVars}}
+    public init({{#allVars}}{{name}}: {{{datatypeWithEnum}}}{{^required}}?{{/required}}{{#hasMore}}, {{/hasMore}}{{/allVars}}) {
+        {{#allVars}}
+        self.{{name}} = {{name}}
+        {{/allVars}}
+    }
+{{/hasVars}}
+{{#additionalPropertiesType}}
+    public var additionalProperties: [String:{{{additionalPropertiesType}}}] = [:]
```

Comment:
This doesn't work with the additionalProperties of the type 'Any' because it outputs two errors.

`Protocol type 'Any' cannot conform to 'Encodable' because only concrete types can conform to protocols`
`Protocol type 'Any' cannot conform to 'Decodable' because only concrete types can conform to protocols`

<details>
  <summary> Here is an example of the generated code</summary>
import Foundation

public struct ProblemDetails: Codable {

    public var type: String?
    public var title: String?
    public var status: Int?
    public var detail: String?
    public var instance: String?

    public init(type: String?, title: String?, status: Int?, detail: String?, instance: String?) {
        self.type = type
        self.title = title
        self.status = status
        self.detail = detail
        self.instance = instance
    }
    public var additionalProperties: [String:Any] = [:]

    public subscript(key: String) -> Any? {
        get {
            if let value = additionalProperties[key] {
                return value
            }
            return nil
        }

        set {
            additionalProperties[key] = newValue
        }
    }

    // Encodable protocol methods

    public func encode(to encoder: Encoder) throws {

        var container = encoder.container(keyedBy: String.self)

        try container.encodeIfPresent(type, forKey: "type")
        try container.encodeIfPresent(title, forKey: "title")
        try container.encodeIfPresent(status, forKey: "status")
        try container.encodeIfPresent(detail, forKey: "detail")
        try container.encodeIfPresent(instance, forKey: "instance")
        try container.encodeMap(additionalProperties) // build error here. `Protocol type 'Any' cannot conform to 'Encodable' because only concrete types can conform to protocols `
    }

    // Decodable protocol methods

    public init(from decoder: Decoder) throws {
        let container = try decoder.container(keyedBy: String.self)

        type = try container.decodeIfPresent(String.self, forKey: "type")
        title = try container.decodeIfPresent(String.self, forKey: "title")
        status = try container.decodeIfPresent(Int.self, forKey: "status")
        detail = try container.decodeIfPresent(String.self, forKey: "detail")
        instance = try container.decodeIfPresent(String.self, forKey: "instance")
        var nonAdditionalPropertyKeys = Set<String>()
        nonAdditionalPropertyKeys.insert("type")
        nonAdditionalPropertyKeys.insert("title")
        nonAdditionalPropertyKeys.insert("status")
        nonAdditionalPropertyKeys.insert("detail")
        nonAdditionalPropertyKeys.insert("instance")
        additionalProperties = try container.decodeMap(Any.self, excludedKeys: nonAdditionalPropertyKeys) // build error here. `Protocol type 'Any' cannot conform to 'Decodable' because only concrete types can conform to protocols`
    }

}
</details>

One possible solution is using [AnyCodable](https://www.github.com/Flight-School/AnyCodable) instead of `Any`